native/acter/src/api/stream.rs

```diff
@@ -112,9 +109,7 @@ impl TimelineStream {
                 let mut edited_content = RoomMessageEventContent::text_markdown(new_msg);
                 edited_content.relates_to = Some(Relation::Replacement(replacement));
 
-                timeline
-                    .send(edited_content.into(), txn_id.as_deref().map(Into::into))
-                    .await;
+                timeline.send(edited_content.into()).await;
```

Comment:
didn't we say we want to replace the timeline.send for edit events with the appropriate [`timeline.edit`](https://matrix-org.github.io/matrix-rust-sdk/matrix_sdk_ui/timeline/struct.Timeline.html#method.edit) ?